scss/mixins/_pagination.scss

```diff
@@ -2,30 +2,8 @@
 
 // scss-docs-start pagination-mixin
 @mixin pagination-size($padding-y, $padding-x, $font-size, $border-radius) {
-  .page-link {
-    padding: $padding-y $padding-x;
-    @include font-size($font-size);
-  }
-
-  .page-item {
-    @if $pagination-margin-start == (-$pagination-border-width) {
-      &:first-child {
-        .page-link {
-          @include border-start-radius($border-radius);
-        }
-      }
-
-      &:last-child {
-        .page-link {
-          @include border-end-radius($border-radius);
-        }
-      }
-    } @else {
-      //Add border-radius to all pageLinks in case they have left margin
-      .page-link {
-        @include border-radius($border-radius);
-      }
-    }
-  }
+  --#{$variable-prefix}pagination-padding: #{$padding-y $padding-x};
+  --#{$variable-prefix}pagination-font-size: #{$font-size};
+  --#{$variable-prefix}pagination-border-radius: #{$border-radius};
```

Comment:
This makes me really enthusiastic regarding how we handle variants. Looks great!